src/main/java/com/revature/beans/User.java

```diff
@@ -238,10 +238,10 @@ public String getPhoneNumber() {
 	public void setPhoneNumber(String phoneNumber) {
 		this.phoneNumber = phoneNumber;
 	}
-	public boolean isDriver() {
+	public boolean isIsDriver() {
```

Comment:
Will this not work with "getIsDriver" ? I know there were some issues with the generated getter but this seems a little redundant

Reply:
getIsDriver appears to work, so I changed it.